builder-support/dockerfiles/Dockerfile.target.centos-10-stream

```
@@ -0,0 +1,19 @@
+# First do the source builds
+@INCLUDE Dockerfile.target.sdist
+
+# This defines the distribution base layer
+# Put only the bare minimum of common commands here, without dev tools
+FROM quay.io/centos/centos:stream10 as dist-base
+
+ARG BUILDER_CACHE_BUSTER=
+
+RUN touch /var/lib/rpm/* && dnf install -y epel-release && \
+    dnf install -y 'dnf-command(config-manager)' && \
+    dnf config-manager --set-enabled crb
```

I just learned there's `/usr/bin/crb enable` as a helper to enable CRB and I think that also works on other EL10-derivatives. It was added in https://src.fedoraproject.org/rpms/epel-release/c/36d5cef26975363fae2dc4704d1cf4d15dc1b315?branch=epel10.

Right now it doesn't make sense but if you'd reuse the same code on a RHEL build it may help:

```suggestion
RUN touch /var/lib/rpm/* && dnf install -y epel-release && \
    dnf install -y 'dnf-command(config-manager)' && \
    /usr/bin/crb enable
```